rasberry_coordination/ServiceRequest service
---
rasberry_coordination/ServiceResponse response
thorvald_base/BatteryArray battery

